v23.1/show-backup.md

```diff
@@ -190,6 +190,33 @@ movr          | public             | vehicles                   | table       |
 . . .
 ~~~
 
+### Show a locality-aware backup
+
+To view a [locality-aware backup](take-and-restore-locality-aware-backups.html), pass locality-aware backup URIs to `SHOW BACKUP`:
+
+{{site.data.alerts.callout_info}}
+`SHOW BACKUP` can display backups taken with the [`incremental_location`](show-backup.html#show-a-backup-taken-with-the-incremental-location-option) option **or** for locality-aware backups, but not for locality-aware backups taken with the `incremental_location` option.
```

Comment:
this was fixed in 23.1

Reply:
noted the fix in 23.1 with a callout to "New in 23.1"